bounties/npm/@hapi/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-06-05",
+    "AffectedVersionRange": "19.1.1",
+    "Summary": "Path traversal in @hapi/hapi via Symlink",
+    "Author": {
+        "Username": "vineetpandey",
+        "Name": "Vineet Pandey"
+    },
+    "Package": {
+        "Registry": "npm",
+        "Name": "@hapi/hapi",
```

Comment:
Replace with just `hapi`

Reply:
Updated it.

Reply:
The latest commit does not show any updates.